dashboard/src/components/DeploymentFormBody/DeploymentFormBody.tsx

```diff
@@ -19,6 +19,7 @@ import Differential from "./Differential";
 import "./Tabs.css";
 
 export interface IDeploymentFormBodyProps {
+  deploymentEvent: DeploymentEvent;
```

Comment:
since the deployment event is now explicit, can you modify here the condition in line 149? There we are guessing if we are deploying or upgrading based on the fact that the `releaseVersion` is set but I prefer the explicit approach.